src/migrations.js

```diff
@@ -24,7 +21,7 @@ require.scopes.migrations = (function() {
 var exports = {};
 exports.Migrations= {
   changePrivacySettings: function() {
-    if (!chrome.extension.inIncognitoContext) {
+    if (!chrome.extension.inIncognitoContext && chrome.privacy ) {
```

Comment:
We would have to rework some things to use spanning mode; however, unless I'm misinterpreting [this](https://developer.chrome.com/extensions/manifest/incognito#spanning) it sounds like we won't be able to make spanning mode work for PB?